mscore/albummanager.h

```diff
@@ -17,35 +17,86 @@
 #include "abstractdialog.h"
 
 namespace Ms {
+class XmlWriter;
 class Movements;
+class MasterScore;
+class Score;
+class AlbumManagerDialog;
+class Album;
+struct AlbumItem;
+
+//---------------------------------------------------------
+//   AlbumManagerItem
+//---------------------------------------------------------
+
+struct AlbumManagerItem : public QObject {
+    Q_OBJECT
+
+public:
+    // can I convert these to references?
```

Comment:
Well, will they ever have a `nullptr` value as pointers? If the answer to that question is 'yes', then the answer is *no*, you can't convert these to references. That's simply because there's no such thing as a null reference.